### DISCUSSION TITLE
Input Shaping Values

### DISCUSSION THREAD

**Discussion by I-wish-2** [mriscoc/Ender3V2S1#916] [Q&A]
Greetings,
In an effort to generate the Input Shaping Values for my E3 Pros which all use Miguel's firmware.  I wanted to use Klipper / RaspberryPI / MainsailOS with a ADXL345 accelerometer in a {stand alone unit} that drives the printer for resonance tests.  I have had success assembling the unit and generating the PNG graph files with give the Frequency values but I'm not sure how to arrive at the Damping values.  I was hoping that someone with Klipper insight might know if the sm (smoothing) values on the graph could be used as the damping values?  I am in contact with the Klipper community as well although the different platforms and values may not be comparable.

For the E3 Pro #2 data is as follows:
for the X axis:  Recommended Shaper = EI  [ EI (94.0 Hz, vibr=1.1%, sm~=0.04, accel<=16500]
for the Y axis:  Recommended Shaper = EI  [ EI (47.6 Hz, vibr=0.8%, sm~=0.14, accel<=4200]

I have attached the graphs as well for more details.  Thanks for reading and any comments are most welcome.
D
![E3 Pro #2 Shaper_Calibrate_X](https://github.com/mriscoc/Ender3V2S1/assets/103605403/8aba741b-5f92-4e90-b39b-7c16af759cfa)
![E3 Pro #2 Shaper_Calibrate_Y](https://github.com/mriscoc/Ender3V2S1/assets/103605403/9e324e3d-e960-4424-af1c-cfcc73a44775)

**Comment by mriscoc** [comment]
Hi, you could ask in the Marlin's Github Input Shaping thread: https://github.com/MarlinFirmware/Marlin/pull/24797

**Reply by I-wish-2** [reply]
Thank yo Miguel for the link and I will give it a try.  I'm sure there is a correlation between Marlins Input Shaping Frequency and Damping and Klipper's Frequency and Shapers values.
Thanks again
D
